### PR TITLE
chore: improve footer version info

### PR DESCRIPTION
**Pull request by septs**

### CODE REVIEW

File: src/_locales/zh/messages.json [hunk 1]
```diff
@@ -261,8 +261,10 @@
     "submit": "提交",
     "unknown_time": "未知时间",
     "username": "用戶名",
-    "version": "版本",
-    "version_link": "https://github.com/DimensionDev/Maskbook/releases#{{tag}}",
+    "version_of_stable": "版本 {{version}}",
+    "version_of_unstable": "版本 {{version}}-{{build}}-{{hash}}",
+    "version_of_release": "https://github.com/DimensionDev/Maskbook/releases/tag/{{tag}}",
```
Comment (Jack-Works) [inline src/_locales/zh/messages.json:266]: Why links need to be put in i18n file?

File: src/extension/options-page/DashboardComponents/FooterLine.tsx [hunk 2]
```diff
@@ -67,21 +67,27 @@ const FooterLink = function (props: React.PropsWithChildren<FooterLinkProps>) {
...
     const { t } = useI18N()
     const classes = useStyles()
-    const { version } = globalThis.browser?.runtime.getManifest() ?? {}
-    const versionLink = t('version_link', { tag: process.env.VERSION })
-
     const [aboutDialog, openAboutDialog] = useModal(DashboardAboutDialog)
-
+    const version = globalThis.browser?.runtime.getManifest()?.version ?? process.env.TAG_NAME.slice(1)
+    const openVersionLink = (event: React.MouseEvent) => {
+        // `MouseEvent.prototype.metaKey` on macOS (<kbd>Command</kbd>), Windows (<kbd>Windows</kbd>)
```
Comment (Jack-Works) [inline src/extension/options-page/DashboardComponents/FooterLine.tsx:73]: what is `<kbd>`👀

**Review by Jack-Works — DISMISSED** [review]

File: src/_locales/zh/messages.json (hunk 1 above)
Comment (septs) [reply src/_locales/zh/messages.json:266]: #816

Comment (Jack-Works) [reply src/_locales/zh/messages.json:266]: But why? No design notes in that issue to explain

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**